### DISCUSSION TITLE
posts/markdown-examples

### DISCUSSION THREAD

**Discussion by giscus** [1024th/blog-giscus#1] [Announcements]
# posts/markdown-examples

A VitePress site

http://localhost:5173/posts/markdown-examples.html

**Comment by 1024th** [comment]
Hello! 🎉

**Test markdown features...**

# H1

followed by some text

## H2

followed by some text

### H3

followed by some text

#### H4

followed by some text

##### H5

followed by some text

###### H6

followed by some text

Auto-detected link: https://giscus.vercel.app

Can render `inline code`, _italics_, **strong**, 😄

> here is blockquote  
> test `inline code` _italics_ **strong** 😄

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis id sem purus, eu commodo tortor. Donec malesuada ultricies dolor a eleifend. In hac habitasse platea dictumst. Vivamus a faucibus ligula. Nullam molestie tristique arcu, eu elementum metus ultricies sed. Aenean luctus congue lectus, vitae semper erat rhoncus non. Nulla facilisi.

Horizontal rule:

* * *

```js
var test \= function this\_is(){
  console.log("some" + colored + "javascript code with javascript syntax highlighting really long");
}
```

[![image link](https://camo.githubusercontent.com/a1d2dda59ca238e496488566d3fd9af0ff0fb64304214c10432358c112735312/68747470733a2f2f6f63746f6465782e6769746875622e636f6d2f696d616765732f68756c615f6c6f6f705f6f63746f64657830332e676966)](https://camo.githubusercontent.com/a1d2dda59ca238e496488566d3fd9af0ff0fb64304214c10432358c112735312/68747470733a2f2f6f63746f6465782e6769746875622e636f6d2f696d616765732f68756c615f6c6f6f705f6f63746f64657830332e676966)

Test from [https://github.com/suan/github-flavored-markdown-test](https://github.com/suan/github-flavored-markdown-test)

Test math renderer error:

${{{¶∆✓¥€¢|π✓©&amp;#//)()(%£€{π√×¶obrigada€¢¥™£®}\\∆×\]✓\\€¢£¥✓\\pelas#@")(""¢¥¥π÷✓\]¶×∆¢π°flores$@&€¢£¥amei////)($#®€¢£¥}💖

When $a \\ne 0$, there are two solutions to $(ax^2 + bx + c = 0)$ and they are  
$$x = {-b \\pm \\sqrt{b^2-4ac} \\over 2a}$$

**The Cauchy-Schwarz Inequality**

$$\\left( \\sum\_{k=1}^n a\_k b\_k \\right)^2 \\leq \\left( \\sum\_{k=1}^n a\_k^2 \\right) \\left( \\sum\_{k=1}^n b\_k^2 \\right)$$
